public/js/actions/navigation.js

```diff
@@ -6,7 +6,7 @@ function willNavigate() {
   return { type: constants.NAVIGATE };
 }
 
-function navigate() {
+function navigated() {
```

Comment:
this might be the most controversial change in the PR.

This came up because i needed to add a reload action, which is triggering a navigation. The pattern setup in the pause actions is that we use present tense for client actions and past tense for server actions e.g. (pause, paused).